### PR TITLE
Update dashboard and index headers

### PR DESCRIPTION
**Pull request by zack**
Just getting sick of the dashboard page.
Updated headers on the index pages to have a create link since I got rid of them on the dashboard.
I make no claims to being a good designer.
If someone else wants to do something else that's cool too.
screen shot 2017-03-25 at 7 37 41 pm
screen shot 2017-03-25 at 7 37 44 pm

### CODE REVIEW

File: app/views/affiliates/index.html.erb [hunk 1]
```diff
@@ -1,6 +1,13 @@
 <p id="notice"><%= notice %></p>
 
-<h1>Affiliates</h1>
+<div class="row index-header">
+  <div class="col-md-8 title">
+    <span>Affiliates</span>
+  </div>
+  <div class="col-md-4 text-right">
+    <%= link_to 'New affiliate', new_affiliate_path %>
```
Comment (xingyif) [inline app/views/affiliates/index.html.erb]: this line already exists in line 44

File: app/views/affiliates/index.html.erb [hunk 2]
```diff
@@ -1,6 +1,13 @@
 <p id="notice"><%= notice %></p>
 
-<h1>Affiliates</h1>
+<div class="row index-header">
+  <div class="col-md-8 title">
+    <span>Affiliates</span>
```
Comment (xingyif) [inline app/views/affiliates/index.html.erb]: Before:
screen shot 2017-03-26 at 01 20 45
After:
screen shot 2017-03-26 at 01 20 58


Emmm, why bother to change the size? We should just use rails magic by calling `<h2> ... </h2>` if you want to simply change the size.

File: app/views/coordinators/index.html.erb [hunk 3]
```diff
@@ -1,4 +1,11 @@
-<h2>Coordinators</h2>
+<div class="row index-header">
+  <div class="col-md-8 title">
+    <span>Coordinators</span>
+  </div>
+  <div class="col-md-4 text-right">
+    <%= link_to 'New coordinator', new_coordinator_path %>
```
Comment (xingyif) [inline app/views/coordinators/index.html.erb]: do we want to put `new coordinator` at the bottom with `back` or on the right corner? we should make sure we do the same for all

File: app/views/students/index.html.erb [hunk 4]
```diff
@@ -1,4 +1,11 @@
-<h2>Students</h2>
+<div class="row index-header">
+  <div class="col-md-8 title">
+    <span>Students</span>
+  </div>
+  <div class="col-md-4 text-right">
+    <%= link_to 'New student', new_student_path %>
```
Comment (xingyif) [inline app/views/students/index.html.erb]: same here. right or bottom

File: app/views/tutors/index.html.erb [hunk 5]
```diff
@@ -1,28 +1,35 @@
-<h2>Tutors</h2>
+<div class="row index-header">
+  <div class="col-md-8 title">
+    <span>Tutors</span>
```
Comment (xingyif) [inline app/views/tutors/index.html.erb]: size

File: app/views/welcome/index.html.erb [hunk 6]
```diff
@@ -1,17 +1,17 @@
-<h1>Welcome</h1>
+<div class="dashboard">
+  <div class='row centered'>
+    <div class='col-md-2 text-center'>
+      <%= link_to 'Students', { controller: 'students' }, class: 'button' %>
```
Comment (xingyif) [inline app/views/welcome/index.html.erb]: I like the buttons. but i think we should keep `Welcome`

File: app/views/affiliates/index.html.erb (hunk 2 above)
Comment (zack) [reply app/views/affiliates/index.html.erb]: That is not what it looks like after my update. I think you may have done something wrong.

**Comment by zack** [comment]
@xingyif I moved them all back to the lower left under the assumption we're going to use @nabeel-'s tables in the future. Having navigation at the bottom of a non-paginated table is bad UX, but once we paginate it's not a big deal.

File: app/views/tutors/index.html.erb (hunk 5 above)
Comment (zack) [reply app/views/tutors/index.html.erb]: Size is fine; see screenshot. I think you may have made a mistake when you pulled to test.

Comment (xingyif) [reply app/views/tutors/index.html.erb]: oh, i see the screeshot! cool

File: app/views/tutors/index.html.erb [hunk 7]
```diff
@@ -46,4 +43,5 @@
   <% end %>
 </table>
 
+<%= link_to 'New tutor', new_tutor_path %> |
```
Comment (xingyif) [inline app/views/tutors/index.html.erb]: 👍

File: db/fixtures/affiliates.rb [hunk 8]
```diff
@@ -1,12 +1,12 @@
...
+# 10.times do
+  # Affiliate.seed do |t|
+    # t.name = Faker::Address.city
+    # t.address = Faker::Address.street_address
+    # t.phone_number = Faker::Base.numerify('(###) ###-####')
+    # t.email = Faker::Internet.email
+    # t.website = Faker::Internet.url('example.com')
+    # t.twitter = Faker::Internet.user_name
+  # end
+# end
```
Comment (xingyif) [inline db/fixtures/affiliates.rb]: why comment out?

**Review by xingyif — APPROVED** [review]

**Pull request merged**